docs/api/index.rst

```diff
@@ -1,16 +1,20 @@
-.. _sec-c-api:
+.. _sec-api:
 
-########
- C API
-########
+.. _sec-api-c:
```

Comment:
Why do we have to "sections" here?

Reply:
Since, the intent is to provide an API section, based on the foundational C API, but not limited to. Thus, it is convenient to be api to refer to the 'api-section" instead of referring explicitly to the C API.

Reply:
Okay, that makes sense